draft-ietf-mops-streaming-opcons.md

```diff
@@ -339,17 +331,26 @@ The causes of unpredictable usage described in {{sec-unpredict}} were more or le
 
 In his talk, Sanjay Mishra {{Mishra}} reported that after the CoViD-19 pandemic broke out in early 2020,
  
-- Comcast's streaming and web video consumption rose by 38%, with their reported peak traffic up 32% overall between March 1 to March 30 {{Comcast}},
+- Comcast's streaming and web video consumption rose by 38%, with their reported peak traffic up 32% overall between March 1 to March 30,
 - AT&T reported a 28% jump in core network traffic (single day in April, as compared to pre stay-at-home daily average traffic), with video accounting for nearly half of all mobile network traffic, while
-social networking and web browsing remained the highest percentage (almost a quarter each) of overall mobility traffic {{ATT}}, and 
-- Verizon reported similar trends with video traffic up 36% over an average day (pre COVID-19) {{Verizon}}.
+social networking and web browsing remained the highest percentage (almost a quarter each) of overall mobility traffic, and 
+- Verizon reported similar trends with video traffic up 36% over an average day (pre COVID-19)}.
 
 We note that other operators saw similar spikes during this time period. Craig Labowitz {{Labovitz}} reported 
 
 - Weekday peak traffic increases over 45%-50% from pre-lockdown levels, 
 - A 30% increase in upstream traffic over their pre-pandemic levels, and
 - A steady increase in the overall volume of DDoS traffic, with amounts exceeding the pre-pandemic levels by 40%. (He attributed this increase to the significant rise in gaming-related DDoS attacks ({{LabovitzDDoS}}), as gaming usage also increased.)
 
+Subsequently, the Inernet Architecture Board (IAB) held a COVID-19 Network Impacts Workshop {{IABcovid}} in November 2020. Given a larger number of reports and more time to reflect, the following observations from the draft workshop report are worth considering.
+
+- Participants describing different types of networks reported different kinds of impacts, but all types of networks saw impacts.
+- Mobile networks saw traffic reductions and residential networks saw significant increases.
+- Reported traffic increases from ISPs and IXPs of 15-20 percent within a few weeks matched the usual traffic growth rate for most years.
```

Comment:
This sentence reads ambiguously to me.  Until I looked at the IAB report I wasn't sure whether it was saying "reported traffic increases turned out to match the usual traffic growth rate" or "the growth over just a few weeks was as big as the usual growth over the course of more than a year, representing a 15-20x surge in growth to land at a new normal that was much bigger than anticipated".

Reply:
I don't think it's ambiguous, but I DO think it's too vague to have any meaning, much less two meanings! 

I started with your phrasing, to end up with 

"- Reported traffic increases from ISPs and IXPs over just a few weeks were as big as the traffic growth over the course of a typical year, representing a 15-20% surge in growth to land at a new normal that was much higher than anticipated."

I also added this line, from the IAB report:

"- At DE-CIX Frankfurt, the world's largest Internet Exchange Point in terms of data throughput, the year 2020 has seen the largest increase in peak traffic within a single year since the IXP was founded in 1995. "

Think this is ready to merge?

Reply:
LGTM.